quaint/src/connector/result_set.rs

```diff
@@ -28,7 +28,7 @@ impl ResultSet {
         }
     }
 
-    #[cfg(any(feature = "sqlite", feature = "mysql"))]
+    #[cfg(any(feature = "sqlite-connector", feature = "mysql-connector"))]
     pub(crate) fn set_last_insert_id(&mut self, id: u64) {
         self.last_insert_id = Some(id);
     }
```

Comment:
Why is that only part of the `-connector` variants?

Reply:
This seems to only be used within these two connectors? Otherwise there will be a unused warning.

Reply:
fixed in https://github.com/prisma/prisma-engines/pull/4095/commits/871c12567978612e62385747c66f9a77648ea17f, allowing dead code only in wasm targets.

Reply:
Sorry, what I meant was, why aren't we compiling this method for the `sqlite` and `mysql` feature flags (the wasm flags) and only the `sqlite-connector` and `mysql-connector`, although `ResultSet` is always compiled?

Reply:
I think I can reply here @Weakky: only the MySQL and SQLite quaint connectors needed to use this `set_last_insert_id(u64)`, which is fundamental for Prisma to support autoincrement IDs when performing `create*` operations on those database systems.

Also, coincidentally, I've already removed feature flags and the `crate` visibility marker in this PR, as I needed to access that method from the `js-connectors` crate: https://github.com/prisma/prisma-engines/pull/4124